### DISCUSSION TITLE
Is there a way to force the temp sensor to use routers?

### DISCUSSION THREAD

**Discussion by JacekKac** [Koenkk/zigbee2mqtt#25151] [General]
Some of temp sensor are constantly droping out of the network - low lqi. There are repeaters in place: 

Tuya TS011F_plug_3
and
IKEA [E1746]

but they are not willing to connect to them instead choosing the coordinator which is fair away. In case of Temp_2 and Temp_3 they are 2 meters away from Tuya plug, and 4 meters away of IKEA. None of them are willing to join them. 

![image](https://github.com/user-attachments/assets/b45a4bc8-d5df-4cad-a63e-098304b03868)

**Comment by chris-1243** [comment]
What kind of sensor are you using (brands, etc...)

I would remove those two sensors from your network and factory reset them.
Now, select the router you wish to use then enable permit join. Then, start the pairing process
